videojs_dev_app/wsgi.py

```diff
@@ -0,0 +1,16 @@
+"""
```

Comment:
Why do we need wsgi.py? I thought videojs_dev_app is supposed to be run in dev mode only.